packages/mask/package.json

```diff
@@ -123,7 +123,12 @@
     "webcrypto-liner": "1.2.4",
     "webextension-polyfill": "^0.8.0",
     "z-schema": "^5.0.2",
-    "zod": "3.11.6"
+    "zod": "3.11.6",
+    "react-icons": "4.3.1",
+    "@google/model-viewer": "1.9.2",
+    "axios": "^0.24.0",
```

Comment:
Please do not add Axios. And you should not send HTTP requests in SNS Adaptor either. It might be blocked by CSP.
Please use `Services.Helper.fetch` (defined in packages/mask/src/extension/background-script/HelperService/index.ts). You may want to change the signature of that function to support POST requests with payloads.